policyengine_us/tests/policy/baseline/gov/states/wv/tax/income/subtractions/senior_citizen_disability_deduction/wv_senior_citizen_disability_deduction_person.yaml

```diff
@@ -0,0 +1,27 @@
+- name: West Virginia senior citizen disability deduction if this person is ineligible with $1,000 total modification and $8,000 subtractions part two.
+  period: 2022
+  input:
+    wv_senior_citizen_disability_deduction_eligible_person: false
+    wv_senior_citizen_disability_deduction_total_modifications: 1_000
+    adjusted_gross_income_person: 20_000
+  output:
+    wv_senior_citizen_disability_deduction_person: 0
+    # min(adjusted_gross_income_person,8000)- wv_total_modification
```

Comment:
```suggestion
    # min(adjusted_gross_income_person - wv_total_modification ,8000)
```

Reply:
I think the logic here 
![image](https://github.com/PolicyEngine/policyengine-us/assets/104268340/ad39e6ee-cf78-4522-b4a0-0a92b87da664)
![image](https://github.com/PolicyEngine/policyengine-us/assets/104268340/dbdf2263-6324-4bd0-9894-17370fdbddf1)
should cap the income at 8000